docs/fastdds/xml_configuration/xml_configuration.rst

```diff
@@ -36,7 +36,7 @@ to create complete XML files.
     /fastdds/xml_configuration/domainparticipant
     /fastdds/xml_configuration/datawriter
     /fastdds/xml_configuration/datareader
-    /fastdds/xml_configuration/library_settings
+    /fastdds/xml_configuration/intra_process_delivery_xml_profile
     /fastdds/xml_configuration/transports
```

Comment:
A warning is being thrown because the file is still named library settings. 
Keep the name of the file as library settings, and leave just here the previous link to the document.